webapp/src/components/VisualBar.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { BoxProps } from "@mui/material";
+import { screen } from "@testing-library/react";
+import { renderWithTheme } from "mocks/utils";
+import VisualBar from "./VisualBar";
+
+const renderVisualBar = (
+  formattedValue: string,
+  value: number,
+  color: BoxProps["bgcolor"]
+) =>
+  renderWithTheme(
+    <VisualBar formattedValue={formattedValue} value={value} color={color} />
+  );
+
+describe("VisualBar", () => {
+  it("should render expected value with correct styling (height, width, bgcolor", () => {
+    renderVisualBar("39.1%", 0.39134, "#00B686");
+    expect(screen.getByText("39.1%")).toBeVisible();
+    expect(
+      screen.getByText("39.1%").parentElement!.lastChild!.firstChild
```

Comment:
We can save `screen.getByText("39.1%")` to a variable instead of repeating it.
```suggestion
    const typography = screen.getByText("39.1%");
    expect(typography).toBeVisible();
    expect(
      typography.parentElement!.lastChild!.firstChild
```